### DISCUSSION TITLE
laragon .test url with Google OAuth

### DISCUSSION THREAD

**Discussion by ylada** [leokhoa/laragon#547] [Q&A] [answered]
I started using Laragon / wordpress and like it! I came across a problem when I need to use Google Calendar API. The .test URL is not accepted by the Google OAuth and showing the following error:

Invalid Origin: must end with a public top-level domain (such as .com or .org).

Is there a way to use GOOGLE APIs on a local development on Laragon? 
Thank you!

**Comment by leokhoa** [comment, marked as answer]
@ylada : You can try using pretty url with .org/.com with Laragon. Just update it in **Menu > Laragon > Preferences**

**Reply by ylada** [reply]
Thank you. It worked, at least on the google side.
